src/wallet-crypto.js

```diff
@@ -321,26 +321,20 @@ function decryptDataWithPassword (data, password, iterations, options) {
   return res;
 }
 
-function stretchPassword (password, salt, iterations, keylen) {
+function stretchPassword (password, salt, iterations, keyLenBits) {
   assert(salt, 'salt missing');
-  assert(password, 'password missing');
-  assert(iterations, 'iterations missing');
-  assert(typeof (sjcl.hash.sha1) === 'function', 'missing sha1, make sure sjcl is configured correctly');
-
-  var hmacSHA1 = function (key) {
-    var hasher = new sjcl.misc.hmac(key, sjcl.hash.sha1); // eslint-disable-line new-cap
-    this.encrypt = hasher.encrypt.bind(hasher);
-  };
-
-  salt = sjcl.codec.hex.toBits(salt.toString('hex'));
-  var stretched = sjcl.misc.pbkdf2(password, salt, iterations, keylen || 256, hmacSHA1);
+  assert(typeof password === 'string', 'password string required');
```

Comment:
probably not important, but the previous assert failed on empty string while the new one doesn't